testing/bootstrap.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Instead of writing your own bootstrap script to handle checking out the code you should consider using [ours](https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py). In addition to checking out the repo at the correct commit and merging the PR changes before testing, our script uploads logs to a GCS bucket. This provides persistence (pod logs are only around until the pod is deleted by sinker) and allows logs and test results to be consumed by https://k8s-gubernator.appspot.com.

You can use the ['execute'](https://github.com/kubernetes/test-infra/blob/master/scenarios/execute.py) bootstrap scenario to run your tests if you move the test commands (looks like the last 3 commands in this file) into a shell script.
To configure bootstrap to use the execute scenario to run your test you will need to add an entry like [this](https://github.com/kubernetes/test-infra/blob/22c184e97746fb8ae98b348e357b0f9d68383263/jobs/config.json#L32-L40) to config.json. Here is the [prowjob config](https://github.com/kubernetes/test-infra/blob/22c184e97746fb8ae98b348e357b0f9d68383263/prow/config.yaml#L5999-L6010) for that same job so you have a complete example to look through.

Reply:
I don't know quite how to evaluate this option. I'll come chat with you about it.

Reply:
So, I think probably the best way to go is to merge this as-is (since it's working locally) and then to rebase it to use your python runner.